omnibus/files/env-customization/windows/env_customization.rb

```diff
@@ -46,9 +47,9 @@
 
   STDERR.puts <<~EOF
 
-    If you would not like ChefDK to try to fix the HOME environment variable,
-    check the CHEFDK_ENV_FIX environment variable. Setting this value to 0
-    prevent this modification to your HOME environment variable.
+    If you would not like Chef Workstation to try to fix the HOME environment
+    variable, check the CHEFDK_ENV_FIX environment variable. Setting this value
```

Comment:
Can we rename this environment variable? I don't think it would be hard but, do we need to update any documentation? 🤔 💭

Reply:
I think this is fine to change because it is mainly used as an implementation detail of the [`start-chefws.ps1`](https://github.com/chef/chef-workstation/blob/829ab43cd996ed00c33cbae7743537b195eacd5c/omnibus/resources/chef-workstation/msi/assets/start-chefws.ps1#L5) script. We can also do it in a backwards compatible way.

Reply:
Im fine with changing it in this PR if you think that will be ok @tyler-ball 

Thanks for your approval! 💯